charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
-
-        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
-
-        <p>Solid defines the concept of storage (one feature of the construct informally referred to as Solid Pods), in which users place data and control access to it, and a suite of protocols for applications to use storages.</p>
-
-        <p>Solid offers several advantages over more traditional architectures for data use by web services today, including:</p>
+        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created them.
+        </p>
 
-        <ul>
-          <li>Storage is controlled by individuals or organizations, and can be delegated to other parties.</li>
-          <li>Interoperability between applications and various storage server implementations of the protocol.</li>
-          <li>A decentralized trust model enabling authentication and authorization mechanisms across the web.</li>
-        </ul>
+        <p><span class=new>A number of initiatives — such as <a href="https://indieweb.org/">IndieWeb</a>, <a href="https://www.w3.org/community/unhosted/">Unhosted</a>, and <a href="https://solidproject.org/">Solid</a> — have emerged to propose alternative models for web applications.
+        They are</span> largely based on existing open web standards, and extend them to give priority to individual and community autonomy.</p>
+        
+        <p>The <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> has been incubating <a href="https://solidproject.org/TR/">specifications</a> since 2018 towards that end, i.e., <q cite="https://www.w3.org/groups/cg/solid">to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces.</q>
+        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/community/fed-id/">Federated Identity CG</a> and the <a href="https://www.w3.org/community/socialcg/">Social Web Incubator CG</a>).</span></p>
```

Comment:
"Official" CG joint meetings included:

```suggestion
        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/groups/cg/credentials/">Credentials Community Group</a>, the <a href="https://www.w3.org/groups/cg/wicg/">Web Platform Incubator Community Group</a>, the <a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator CG</a>).</span></p>
```

Reply:
thanks for the complement. Why remove the Federated Identity CG, though? I was pretty sure there was some interactions -- maybe not a dedicated joint meeting, though?

Reply:
Short: If you feel that mentioning the CG will be more appealing in this charter since they are also proposing a WG, go for it. I'd suggest to keep Credentials CG though.

Extended: Right. It was because of the initial joint meeting. If my memory serves me well, later that year some of the folks in WICG started FedID CG. No doubt that there are many individuals interacting randomly with various Groups. (You don't want to know where I've been :P )

Reply:
I started joining FedId meetings, and the work done there is relevant. There is also https://github.com/w3c/strategy/issues/427#issuecomment-1768348549
Solid-OIDC has an identity provider, so we do have an overlap there, which we should keep tracking.

Reply:
```suggestion
        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/groups/cg/credentials/">Credentials Community Group (CCG)</a>, the <a href="https://www.w3.org/community/fed-id/">Federated Identity Community Group</a>, the <a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator Community Group</a>, and the <a href="https://www.w3.org/groups/cg/wicg/">Web Platform Incubator Community Group (WICG)</a>).</span></p>
```

Reply:
We had @elf-pavlik and others come to the FedID CG a few times and we welcome the engagement here:

https://github.com/fedidcg/FedCM/issues/240#issuecomment-1782858001

I'm not sure if it is good or bad to make the association with the FedID CG for this group, but in case it helps, I think it would be fair to say that "it has regularly sought cross-fertilization with the FedID CG", based on this single thread I pointed to above.